src/modules/hub.bicep

```
@@ -0,0 +1,75 @@
+/**
+ * Parameters
+ */
+
+@description('Name of the hub. Used to ensure unique resource names.')
+param hubName string
+
+// Generate unique storage account name
+var storageAccountSuffix = 'store'
+var storageAccountName = '${substring(replace(toLower(hubName), '-', ''), 0, 24 - length(storageAccountSuffix))}${storageAccountSuffix}'
+
+@description('Specifies the location for resources. See https://aka.ms/azureregions.')
+param location string = resourceGroup().location
+
+@allowed([
+  'Premium_LRS'
+  'Premium_ZRS'
+])
+@description('Optional. Storage account SKU. LRS = Lowest cost, ZRS = High availability. Note Standard SKUs are not available for Data Lake gen2 storage.')
+param storageSku string = 'Premium_LRS'
+
+@description('Optional. Tags for all resources.')
```

Aligning to the spec:
```suggestion
@description('Optional. Tags to apply to all resources. We will also add the cm-resource-parent tag for improved cost rollups in Cost Management.')
```